generators/server/templates/dotnetcore/src/Project.Domain/Project.csproj.ejs

```diff
@@ -0,0 +1,37 @@
+<%#
+ Copyright 2013-2020 the original author or authors from the JHipster project.
+ This file is part of the JHipster project, see https://www.jhipster.tech/
+ for more information.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netstandard2.1</TargetFramework>
+        <RootNamespace><%= namespace %>.Domain</RootNamespace>
+        <OutputType>Library</OutputType>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.AspNetCore.Identity" Version="2.2.0" />
+        <%_ if(databaseType === 'oracle') { _%>
+        <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="2.2.0" />
+        <%_ } else { %>
+        <PackageReference Include="Microsoft.AspNetCore.Identity.EntityFrameworkCore" Version="3.1.5" />
+        <%_ } %>
+        <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+    </ItemGroup>
+
+    <ItemGroup>
+        <ProjectReference Include="..\JhipsterSampleApplication.Crosscutting\JhipsterSampleApplication.Crosscutting.csproj" />
```

Comment:
Use variable

Reply:
> `While following the contributing.md it recommended squashing commits and doing a git push -f so that was I was using it.`
> Can you tell me where ? There is two reasons to force push, first if you do a rebase and second if you squash for clean git history but it's before submit PR

Rebase and push -f mentioned at 
https://github.com/jhipster/jhipster-dotnetcore/blob/master/CONTRIBUTING.md#submitting-a-pull-request


And squash mentioned at 
https://github.com/jhipster/jhipster-dotnetcore/blob/master/CONTRIBUTING.md#-git-commit-guidelines 
might also imply a push -f if changes were pushed to a remote repository before

> 
> you still have two points to address and i think it's ok.
> 
> You have done a fantastic job, thank you again !

Thank you @nicolas63 for all the support on this project